projects/admin/src/app/api/ill-request-api.service.ts

```diff
@@ -48,7 +48,7 @@ export class IllRequestApiService extends BaseApi {
     const query = `patron.pid:${patronPid}`;
     return this._recordService
       .getRecords(this.RESOURCE_NAME, query , 1, RecordService.MAX_REST_RESULTS_SIZE,
-                  undefined, filters, BaseApi.reroJsonheaders, 'created')
+                  undefined, filters, BaseApi.reroJsonheaders, '-created')
```

Comment:
Add parameter into the function like this `getByPatronPid(patronPid: string, filters?: {[key: string]: string}, order: string = '-created')`
and into the fonction getRecords `BaseApi.reroJsonheaders, $order`